### PR TITLE
chore(deps): Update @sentry/types for 7.2.0

### PR DESCRIPTION
**Pull request by yog27ray**
## Changes

Update sentry function "setupOnce" declaration.
...

## Checklist
- [x] Tests for new code (see [advice on the tests we use](https://github.com/PostHog/posthog-js#tiers-of-testing))
- [x] Accounted for the impact of any changes across different browsers
- [x] TypeScript definitions (module.d.ts) updated and in sync with library exports (if applicable)

### CODE REVIEW

File: src/module.d.ts [hunk 1]
```diff
@@ -851,7 +851,7 @@ declare namespace posthog {
 export class SentryIntegration implements Integration {
     constructor(posthog: posthog, organization?: string, projectId?: number, prefix?: string)
     name: string
-    setupOnce(addGlobalEventProcessor: (callback: EventProcessor) => void): void
+    setupOnce(addGlobalEventProcessor: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void
```
Comment (Twixes) [inline src/module.d.ts:854]: Hm, I don't think this is quite correct as we don't support `getCurrentHub` in https://github.com/PostHog/posthog-js/blob/7d5eefb21fb7634ae1a74d0fd099e730567aea89/src/posthog-core.js#L1493.

What's the intention with adding this and updating types to 7.2.0?

**Comment by mariusandra** [comment]
> @mariusandra Please review.

and why me? :D

**Comment by yog27ray** [comment]
> > @mariusandra Please review.
> 
> and why me? :D

File: src/module.d.ts (hunk 1 above)
Comment (yog27ray) [reply src/module.d.ts:854]: @Twixes  in the latest release of [@sentry/types](https://www.npmjs.com/package/@sentry/types) 7.x.x, an additional parameter has been added in setupOnce function i.e. getCurrentHub. So due to this, I am not able to update this library to the latest version as I am using sentry postHog integration also. This update will to keep both sentry and postHog libraries to the latest versions.

**Review by Twixes — APPROVED** [review]
Looks good, thanks

**Pull request merged**